packages/react-router/modules/Route.js

```diff
@@ -37,23 +35,28 @@ class Route extends React.Component {
   getChildContext() {
     return {
       router: {
-        ...this.context.router,
```

Comment:
@timdorr  I was reading this code to understand how Router works, I see that `Router` already puts the `router` on the `context`. Why would `Route` also do the same?

Reply:
It's so it can attach the `match` to the context for any children under this section of the tree.